libs/client/cornflow_client/constants.py

```diff
@@ -90,6 +90,91 @@
     pl.LpStatusUndefined: STATUS_UNDEFINED,
 }
 
+PARAMETER_SOLVER_TRANSLATING_MAPPING = {
+    ('bar_iter_limit', 'pyomo', 'cbc') : 'barr' ,
+    ('best_obj_stop', 'pyomo', 'cbc') : 'primalT' ,
+    ('cutoff', 'pyomo', 'cbc') : 'cuts' ,
+    ('iteration_limit', 'pyomo', 'cbc') : 'maxIt' ,
+    ('solution_limit', 'pyomo', 'cbc') : 'maxSaved' ,
+    ('time_limit', 'pyomo', 'cbc') : 'sec' ,
+    ('abs_gap', 'pyomo', 'cbc') : 'allow' ,
+    ('pump_passes', 'pyomo', 'cbc') : 'pumpC' ,
+    ('heuristics', 'pyomo', 'cbc') : 'heur' ,
+    ('rel_gap', 'pyomo', 'cbc'): 'ratio' ,
+
+    ('is_mip', 'pulp', 'cbc') : 'mip' ,
+    ('abs_gap', 'pulp', 'cbc') : 'gapAbs' ,
+    ('cutoff', 'pulp', 'cbc') : 'cuts' ,
+    ('time_limit', 'pulp', 'cbc') : 'timeLimit' ,
+    ('threads', 'pulp', 'cbc') : 'threads' ,
+    ('presolve', 'pulp', 'cbc') : 'presolve' ,
+    ('rel_gap', 'pulp', 'cbc') : 'gapRel' ,
+    
+
+    ('bar_iter_limit', 'pyomo', 'gurobi') : 'BarIterLimit' ,
+    ('best_obj_stop', 'pyomo', 'gurobi') : 'BestObjStop' ,
+    ('cutoff', 'pyomo', 'gurobi') : 'Cutoff' ,
+    ('iteration_limit', 'pyomo', 'gurobi') : 'IterationLimit' ,
+    ('mem_limit', 'pyomo', 'gurobi') : 'MemLimit' ,
+    ('solution_limit', 'pyomo', 'gurobi') : 'SolutionLimit' ,
+    ('time_limit', 'pyomo', 'gurobi') : 'TimeLimit' ,
+    ('abs_gap', 'pyomo', 'gurobi') : 'MIPGapAbs' ,
+    ('rel_gap', 'pyomo', 'gurobi') : 'MIPGap' ,
+    ('branch_dir', 'pyomo', 'gurobi') : 'BranchDir' ,
+    ('pump_passes', 'pyomo', 'gurobi') : 'PumpPasses' ,
+    ('heuristics', 'pyomo', 'gurobi') : 'Heuristics' ,
+    ('presolve', 'pyomo', 'gurobi') : 'Presolve' ,
+    ('threads', 'pyomo', 'gurobi') : 'threads' ,
+    
+    ('is_mip', 'pulp', 'gurobi') : 'mip' ,
+    ('time_limit', 'pulp', 'gurobi') : 'timeLimit' ,
+    ('presolve', 'pulp', 'gurobi') : 'presolve' ,
+    ('abs_gap', 'pulp', 'gurobi') : 'gapAbs' ,
+    ('feasibility_tol', 'pulp', 'gurobi') : 'FeasibilityTol' ,
+    ('iteration_limit', 'pulp', 'gurobi') : 'IterationLimit' ,
```

Comment:
Could we have a `rel_gap` option for (pulp, gurobi)?

Reply:
Yes, and I will include it. However, it only works for the CMD version. Should we make a clear statement?